packages/kbn-check-mappings-update-cli/src/run_check_mappings_update_cli.ts

```diff
@@ -6,95 +6,44 @@
  * Side Public License, v 1.
  */
 
-import deepEqual from 'fast-deep-equal';
 import { run } from '@kbn/dev-cli-runner';
-import { createTestEsCluster } from '@kbn/test';
-
-import { extractMappingsFromPlugins } from './extract_mappings_from_plugins';
-
-import { checkAdditiveOnlyChange } from './check_additive_only_change';
-import { checkIncompatibleMappings } from './check_incompatible_mappings';
-import { readCurrentMappings, updateCurrentMappings } from './current_mappings';
+import { runMappingsCompatibilityChecks } from './compatibility';
+import { runModelVersionMappingAdditionsChecks } from './mappings_additions';
 
 run(
   async ({ log, flagsReader, addCleanupTask }) => {
     const fix = flagsReader.boolean('fix');
     const verify = flagsReader.boolean('verify');
+    const task = flagsReader.string('task');
 
-    /**
-     * Algorithm for checking compatible mappings. Should work in CI or local
-     * dev environment.
-     * 1. Extract mappings from code as JSON object
-     * 2. Check if extracted mappings is different from current_mappings.json, current_mappings.json stores
-     *    the mappings from upstream and is commited to each branch
-     * 3. Start a fresh ES node
-     * 4. Upload current_mappings.json to ES node
-     * 5. Upload extracted mappings.json to ES node
-     * 6. Check result of response to step 5, if bad response the mappings are incompatible
-     * 7. If good response, write extracted mappings to current_mappings.json
-     */
-
-    log.info('Extracting mappings from plugins');
-    const extractedMappings = await log.indent(4, async () => {
-      return await extractMappingsFromPlugins(log);
-    });
-
-    const currentMappings = await readCurrentMappings();
-    const isMappingChanged = !deepEqual(currentMappings, extractedMappings);
-
-    if (!isMappingChanged) {
-      log.success('Mappings are unchanged.');
-      return;
-    }
-
-    if (verify) {
-      log.info('Checking if any mappings have been removed');
+    if (!task || task === 'mapping-addition') {
+      log.info('Running model version mapping addition checks');
```

Comment:
Updated the existing package / cli command to run both checks, given it makes more sense to execute everything in a single CI task.